client/src/i18n/en-US.json

```
@@ -892,11 +892,15 @@
       "heading": "Submission Content",
       "upload": {
         "label": "Upload",
-        "caption": "Select a supported file to be imported."
+        "caption": "Select a supported file to be imported.",
+        "file_caption": "Accepted file types: ",
+        "file_types": ".csv, .docbook, .docx, .epub, .html, .latex, .md, .odt, .rtf"
```

I just double-checked by testing it and `.txt` files are supported as input types despite [the findings I documented here](https://github.com/MESH-Research/Pilcrow/issues/517#issuecomment-1105478040). I don't know why I documented they were output only. Maybe that was the case at the time and it changed since then? In any case, I apologize for the confusion. 
```suggestion
        "file_types": ".csv, .docbook, .docx, .epub, .html, .latex, .md, .odt, .rtf, .txt"
```